syft/sandbox.py

```diff
@@ -164,5 +164,5 @@ def distribute_dataset(data, workers):
     print("Done!")
 
 
-def hook(gbs):
+def make_hook(gbs):
```

Comment:
It is not necessarily a bug, but it might cause a name collision when import hook from syft (in case we do not have the hook initialized syft.hook will be a function)